src/lib/services/wasm/contract/lcd.ts

```diff
@@ -150,3 +151,13 @@ export const getInstantiatedContractsByAddressLcd = (
         label: "",
       }));
     });
+
+export const getContractCw2InfoLcd = async (
+  endpoint: string,
+  contractAddress: BechAddr32
+) =>
+  axios
+    .get(
+      `${endpoint}/cosmwasm/wasm/v1/contract/${encodeURI(contractAddress)}/raw/Y29udHJhY3RfaW5mbw%3D%3D`
+    )
+    .then(({ data }) => JSON.parse(libDecode(data.data)) as ContractCw2Info);
```

Comment:
Maybe we should create Zod type for `ContractCw2Info`

Reply:
555555 will check